opm/utility/ECLGraph.hpp

```diff
@@ -181,6 +182,43 @@ namespace Opm {
         std::vector<double>
         flux(const PhaseIndex phase) const;
 
+        /// Retrieve result set vector from current view (e.g., particular
+        /// report step) linearised on active cells.
+        ///
+        /// \tparam T Element type of result set vector.
+        ///
+        /// \param[in] vector Name of result set vector.
+        ///
+        /// \return Result set vector linearised on active cells.
+        template <typename T>
+        std::vector<T>
+        rawLinearisedCellData(const std::string& vector) const;
+
+        /// Convenience type alias for \c UnitSystem PMFs.
```

Comment:
Pointer to member function.  It's a fairly common abbreviation, but I'll add an explanation.